java/integTest/build.gradle

```diff
@@ -11,6 +11,7 @@ dependencies {
     implementation project(':client')
 
     implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.13.0'
+    implementation 'com.google.code.gson:gson:2.10.1'
```

Comment:
is the license ok?

Reply:
Apache-2
Anyway it is used for tests only.

Reply:
I'll align with you to use this google library instead of `org.skyscreamer.jsonassert` for JSON equality test in my next PR then. We don't need 2 different lib for the same purpose.